src/support/isEntityType.ts

```diff
@@ -0,0 +1,6 @@
+import { Buildable, Constructor } from './Type';
+import { Entity } from '../Entity';
+
+export function isEntityType(buildable: Buildable): buildable is Constructor<Entity> {
+    return buildable?.prototype instanceof Entity;
```

Comment:
make sure this function is tested – I know it's tiny, but it's good to have a sanity check on these simple ones too